### PR TITLE
Update Liking functions

### PR DESCRIPTION
**Pull request by YcMcGong**
This commit updates the Liking function

### CODE REVIEW

File: SmileyBackend/SQL_helper.py [hunk 1]
```diff
@@ -170,4 +170,27 @@ def show_all_friends(email, cursor):
...
+        cursor.execute("""COMMIT""")
+
+def fetch_like(user_email, attraction_url, cursor):
+    # Look up if a like exist
+    cursor.execute("""SELECT user_email, attraction_url, rating 
+    FROM Likes 
+    WHERE user_email = %s 
+    AND attraction_url = %s""",(user_email, attraction_url))
+    data = cursor.fetchone()
+    print(data)
```
Comment (BangtianZhou) [inline SmileyBackend/SQL_helper.py:195]: Please delete the print

File: SmileyBackend/SQL_helper.py [hunk 2]
```diff
@@ -170,4 +170,27 @@ def show_all_friends(email, cursor):
...
+# Likes
+class Like():
+    def __init__(self, user_email, attraction_url, rating):
+        self.user_email = user_email
+        self.attraction_url = attraction_url
+        self.rating = rating
+
+def add_like(user_email, attraction_url, rating, cursor):
+    if not fetch_like(user_email, attraction_url, cursor):
+        print (fetch_like(user_email, attraction_url, cursor))
```
Comment (BangtianZhou) [inline SmileyBackend/SQL_helper.py:184]: please delete the print

**Review by BangtianZhou — CHANGES_REQUESTED** [review]

File: SmileyBackend/SQL_helper.py (hunk 2 above)
Comment (YcMcGong) [reply SmileyBackend/SQL_helper.py:184]: Sure!

File: SmileyBackend/SQL_helper.py (hunk 1 above)
Comment (YcMcGong) [reply SmileyBackend/SQL_helper.py:195]: Sure!

**Pull request closed without merging**